README.md

```diff
@@ -40,7 +41,7 @@ Find out more:
 - [Blog](https://emberjs.com/blog)
 - [Builds](https://emberjs.com/builds)
 
-# Contribution
+## Contributions
```

Comment:
Thanks for your contribution. Can you keep just this change and remove the others?

Reply:
sure
Done !